wordpress.org/public_html/wp-content/themes/pub/wporg-openverse/README.md

```diff
@@ -20,6 +21,12 @@ Follow these steps to set up a local playground for the theme:
         ```bash
         $ npm install
         ```
+        If you face issues installing Sass, try the following command, and
+        `npm install` again.
+        ```bash
+        $ npm install node-sass@npm:sass
```

Comment:
Does this command replace `node-sass` with `sass` when running `npm install`? I was manually changing the `package.json` to get it to build before.

Reply:
Yeah, I don't know the specifics about that package but `node-sass` has problems running on M1 while `sass` works. Since this is only needed in dev, best to just use `sass` to get the base theme to build.